site/faq.md

```diff
@@ -300,3 +300,32 @@ your case.
 You will need to use the (experimental) command-line flag
 `--k8s-namespace-whitelist` to enumerate the namespaces that Flux
 attempts to scan for workloads.
+
+### Can I temporarily make flux ignore a deployment?
+
+Yes. The easiest way to do that is to use the following annotation:
+
+```yaml
+    flux.weave.works/ignore: true
+```
+
+A live example of this can be seen
+[here](https://github.com/stefanprodan/openfaas-flux/blob/master/secrets/openfaas-token.yaml).
+
+Please note that not all resources are tracked by flux at the moment,
+we are currently limited to resources of the type `namespace` and the
+set of controllers in
+[resourcekinds.go](https://github.com/weaveworks/flux/blob/master/cluster/kubernetes/resourcekinds.go),
+namely `deployment`, `daemonset`, `cronjob`, `statefulset` and `deployment`).
+
+To stop ignoring these annotated resources, you simply remove the
+annotation from the manifests in git.
+
+If the annotations are carried in the cluster as well (this should
+not happen by default), you need to remove them from the cluster as well,
```

Comment:
Annotating the resource in the cluster is an intended mode of use (it can be a bit easier than committing a change to git).